lib/links.rb

```diff
@@ -6,15 +6,16 @@ def initialize(url)
   end
 
   def run
-    ary = Parser.new(@url).run
+    links = Parser.new(@url).run
 
-    ary.each do |a|
-        response = HTTParty.get(a)
-        if(response.code == 200)
-            puts response.code.to_s.green + "\t" + a
-        else
-            puts response.code.to_s.red + "\t" + a
-        end
+    links.each do |l|
+      response = HTTParty.get(l)
+
+      if (response.code == 200)
+        puts response.code.to_s.green + "\t" + l
+      else
+        puts response.code.to_s.red + "\t" + l
+      end
```

Comment:
You can condense this.

```ruby
colored_code = response.code == 200 ? response.code.to_s.green : response.code.to_s.red
puts colored_code + "\t" + l
```

Reply:
Hhmm... the style guide "told me" to avoid ternary operators. I know they are just suggestions, but yeah, it looks cleaner. I guess I'll use it here

Reply:
actually yes, the ternary operator is to be avoided, mostly

Since we're on this topic (which I wouldn't be too picky about to be honest, but you guys started it xD), I don't particularly like either solutions here.

FYI: "condense this" is not a very good guideline. You generally don't want to condense, but to make it look & read better. Smaller code is not necessarily better looking (wink wink Perl)

Between the two, I'd really go with the original multiline `if`, just adapted since you're putting too many responsibilities in the same place:

```ruby
  # ...
  puts "#{format_code(response.code)}\t#{l}"
  # ...


# down below, or even in a different class
def format_code(code) do
  if (200..299).include?(code)
    code.to_s.green
  else
    code.to_s.red
  end
end
```

also note that I'm using green for any 2XX code, not just 200. Not sure how the original repo does it, but it seems to make more sense, since they're all success responses according to the spec